addons/mail_bot/static/src/js/mailbot_service.js

```diff
@@ -13,7 +14,9 @@ var MailBotService =  AbstractService.extend({
      */
     start: function () {
         this._hasRequest = (window.Notification && window.Notification.permission === "default") || false;
-        if ('odoobot_initialized' in session && ! session.odoobot_initialized) {
+        var lastRequestTime = localStorage.getItem('odoobot_notification_last_request_time');
```

Comment:
Make sure the timestamp you use is universal, that is in UTC i guess.
Use some kind of variation of session.getTZOffset

Reply:
Not sure this is useful. We are interested in a difference between 2 datetimes; whether they are in UTC or TZ-aware is not relevant as long as they are consistent. In this case, `new Date().getTime()` always returns the UTC milliseconds, so it looks good.